pages/common/dolt-init.md

```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-config>.
```

Comment:
This link doesn't work for me
```suggestion
> More information: <https://docs.dolthub.com/cli-reference/cli#dolt-config>.
```

Reply:
Shouldn't it be 
```suggestion
> More information: <https://docs.dolthub.com/cli-reference/cli#dolt-init>.
```

Reply:
you are totally right, I've got it wrong with my other branch, fixed in 2b940581aa22cd07b8b4b2703dd5e3daabeb8b4a